en/docs/develop/troubleshooting/errors-and-stack-traces.md

```
@@ -0,0 +1,226 @@
+
+### Read a compiler diagnostic
+
+A compiler diagnostic uses the form:
+
+```bash
+ERROR [<file>.bal:(<start_line>:<start_col>,<end_line>:<end_col>)] <message>
+```
+
+For example:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)